pandas/core/arrays/sparse.py

```diff
@@ -1262,12 +1262,8 @@ def searchsorted(self, v, side="left", sorter=None):
             v, side, sorter
         )
 
-    def copy(self, deep=False):
-        if deep:
-            values = self.sp_values.copy()
-        else:
-            values = self.sp_values
-
+    def copy(self):
+        values = self.sp_values.copy()
         return self._simple_new(values, self.sp_index, self.dtype)
```

Comment:
@TomAugspurger do we need to copy the index here?

Reply:
Not really sure. I don't know if they're designed for sharing across instances, or whether we mutate them inplace.

Reply:
In any case in sparse.py, we don't mutate them inplace. I would also assume it is fine not to copy them.